examples/chat_app/interface/hddr_llm_onnx_dml_interface.py

```diff
@@ -0,0 +1,182 @@
+import gc
+import os
+import sys
+import onnxruntime_genai as og
+from app_modules.utils import convert_to_markdown, is_stop_word_or_prefix, shared_state
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(current_dir, "..", "..", ".."))
+
+
+class ONNXModel():
+    """A wrapper for OnnxRuntime-GenAI to run ONNX LLM model."""
+
+    def __init__(self, model_path):
+        self.og = og
+        self.model = og.Model(f'{model_path}')
+        self.tokenizer = og.Tokenizer(self.model)
+        self.tokenizer_stream = self.tokenizer.create_stream()
+        self.is_phi = "phi" in model_path
```

Comment:
not for this PR: GenAI needs to have an api to query the model information

Reply:
That could be helpful, instead of checking the model folder name.